gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v3_15_20/schema.yml

```diff
@@ -2,6 +2,7 @@ databaseChangeLog:
   - changeSet:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.

Reply:
It looks to work @a-cordier  🎉